.eslintrc.json

```diff
@@ -0,0 +1,97 @@
+{
+    "extends": ["airbnb-base", "airbnb-typescript/base"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "./tsconfig.json"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
```

Comment:
Same question. Why do we need so many extra rules?

Reply:
The rules that say "error" can be deleted later, as they are enforced by airbnb already. 
The rules that are off is overriding the airbnb rules and turning them off.